actions/build-lzo

```diff
@@ -36,7 +36,7 @@ def build_binary(builder):
     builder.build_binary(build_command)
     os.makedirs(builder.workspace + '/share/hadoop/common/lib')
     os.makedirs(builder.workspace + '/lib/native')
-    copyfile(builder.workspace + '/target/hadoop-lzo-' + LZO_VERSION + '.jar', builder.workspace + '/share/hadoop/common/lib/hadoop-lzo-' + LZO_VERSION + '.jar')
+    copyfile(builder.workspace + '/target/hadoop-lzo-' + LZO_VERSION + '.jar', builder.workspace + '/lzo/share/hadoop/common/lib/hadoop-lzo-' + LZO_VERSION + '.jar')
```

Comment:
It should be:
copyfile(builder.workspace + '/lzo/target/hadoop-lzo-' + LZO_VERSION + '.jar', ..............

The lzo missing is on the location where the jar is created